tour_model_eval/viz_bins_clusters_above_cutoff.ipynb

```diff
@@ -0,0 +1,392 @@
+    "import emission.analysis.modelling.tour_model.featurization as featurization\n",
+    "import viz_bin_cluster as viz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "minus-norfolk",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logger = logging.getLogger()\n",
+    "logger.setLevel(logging.DEBUG)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "configured-tobacco",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# copied from mode_purpose_share.ipynb\n",
+    "# We select participants from real data to be users at this time\n",
+    "participant_uuid_obj = list(edb.get_profile_db().find({\"install_group\": \"participant\"}, {\"user_id\": 1, \"_id\": 0}))\n",
+    "all_users = [u[\"user_id\"] for u in participant_uuid_obj]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "informative-mitchell",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logging.debug('There are %s users, we will plot the graphs for one of them' % len(all_users))\n",
+    "\n",
+    "user = all_users[1]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "particular-inspector",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we should experiment with different values here\n",
+    "radius = 300"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dangerous-galaxy",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trips = pipeline.read_data(uuid=user)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "empirical-projector",
+   "metadata": {},
+   "source": [
+    "## Visualizing trips from bins above cutoff"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dangerous-broadcast",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bin_trips, bins = pipeline.remove_noise(trips, radius)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "little-squad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logging.debug('The list of bins is %s' % bins)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "mexican-intermediate",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "viz.bins_map(bins,trips)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "altered-battery",
+   "metadata": {},
+   "source": [
+    "## Visualizing clusters above cutoff"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "played-gabriel",
+   "metadata": {},
+   "source": [
+    "Note: should run \"Trips from bins above the cutoff only\" first"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "frozen-modern",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# copied from the cluster pipeline code so we can work with the featurization code directly\n",
```

Comment:
since this is above the cutoff, I think we can just use the `pipeline.cluster` function directly. I don't see us using the `feat` object later, only `feat.clusters, feat.labels, feat.data` which is what the `pipeline.cluster` function returns.